### PR TITLE
VCDM v2 EAD + other fixes.

### PR DESCRIPTION
**Pull request by davidlehn**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,11 @@
 ### Added
 - Added a new Verifiable Credential v2.0 version of the Utopia Naturalization
   Certificate.
+- Added a new Verifiable Credential v2.0 version of the Utopia Employment
```
Comment (BigBlueHat) [inline CHANGELOG.md:8]: I added my newer change in #69 in front of the older change on line 7. You added yours below. Should we pick an ordering? I realize this file doesn't care...but the commit log will differ from changelog order...if that matters.

File: credentials/prc/credential.json [hunk 2]
```diff
@@ -3,7 +3,7 @@
     "https://www.w3.org/2018/credentials/v1",
     "https://w3id.org/citizenship/v1"
   ],
-  "id": "https://issuer.oidp.uscis.gov/credentials/83627465",
+  "id": "https://issuer.example.gov/credentials/83627465",
```
Comment (BigBlueHat) [inline credentials/prc/credential.json:6]: 👍

File: credentials/utopia-ead-vc-v2/queries.json [hunk 3]
```diff
@@ -0,0 +1,23 @@
+{
+  "default": {
+    "type": "QueryByExample",
+    "credentialQuery": {
+      "reason": "Please present your Employment Authorization Document Verifiable Credential(s) to complete the verification process.",
+      "example": {
+        "@context": [
+          "https://www.w3.org/ns/credentials/v2",
+          "https://w3id.org/citizenship/v4rc1",
+          "https://w3id.org/vc/render-method/v2rc1"
```
Comment (BigBlueHat) [inline credentials/utopia-ead-vc-v2/queries.json:10]: Do we care what version of render method was used when querying for the credential?

```suggestion
```

I wouldn't think so.

File: credentials/utopia-natcert-vcdm-v2/credential.json [hunk 4]
```diff
@@ -2,7 +2,6 @@
   "@context": [
     "https://www.w3.org/ns/credentials/v2",
     "https://w3id.org/citizenship/v4rc1",
-    "https://w3id.org/security/data-integrity/v2",
```
Comment (BigBlueHat) [inline credentials/utopia-natcert-vcdm-v2/credential.json:5]: Good catch. It'd be nice to have a tool that shows which contexts have no terms from them used--or better yet, which terms came from which context file.

**Review by BigBlueHat — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (davidlehn) [reply CHANGELOG.md:10]: Honestly, I put it at the end to not conflict when merging with your change.  It's not a big deal either way.

**Review by BigBlueHat — APPROVED** [review]

**Pull request merged**